### DISCUSSION TITLE
Outreachy contribution phase

### DISCUSSION THREAD

**Discussion by NgweBecky** [camicroscope/caMicroscope#679] [Q&A]
Greetings to everyone, I got approved in the outreachy application phase and will like to efficiently participate and contribute to caMicroscope community. Please I will like to have suggestions on where to start for the contribution phase, thank you.

**Comment by birm** [comment]
Thanks for the congratulations and for your interest!

I've put a list of some possible good contributor tasks here (https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md)

Please let me know if any are unclear, or if you have any feedback on this list!

**Reply by NgweBecky** [reply]
Thanks, will look into that right away

**Reply by maha-sachin** [reply]
Thanks for sharing the list @birm will follow that

**Comment by valentina-buoro** [comment]
Hello everyone 👋🏼, I'm Valentina Buoro, an outreachy applicant for May 2024 and I'm looking forward to contributing to caMicroscope

**Comment by mhdamaan79** [comment]
👋 Greetings all! I'm Amaan, an Outreachy applicant for May 2024. Excited to dive into caMicroscope's contribution journey!

**Reply by SisiVero** [reply]
Hi @mhdamaan79 , you're welcome

**Comment by amehclibton** [comment]
Hello everyone, I am Clinton Ameh an Outreachy applicant and I am happy to join this community.

**Reply by SisiVero** [reply]
Hi Clinton, it's good to have you

**Comment by maha-sachin** [comment]
Hello everyone, I am Maha an Outreachy applicant for May 2024. I am excited to contribute to creating this standalone executable and ensuring that it aligns with the project's goals and standards.

**Reply by SisiVero** [reply]
Hi Maha, you're welcome

**Comment by Ebere330** [comment]
Hello everyone, I am Ebere an Outreachy applicant for May 2024. I am happy to contribute to creating this standalone  project

**Reply by SisiVero** [reply]
Hi Ebere, you're welcome

**Comment by shreyarustagi** [comment]
Hi everyone! My name is Shreya, and I'm an outreachy applicant for May 2024 as well. Super excited to contribute to caMicroscope and collaborate with everyone!

**Reply by NgweBecky** [reply]
welcome everyone, I look forward to collaborating and learning from you

**Reply by SisiVero** [reply]
Hi Shreya, you're welcome

**Comment by Audiostrap** [comment]
Hi everyone I’m Joy an outreachy applicant looking to understand the internship and make tangible contributions

**Reply by SisiVero** [reply]
Hello Joy, it's good to have you.

**Comment by Kingjhenny** [comment]
Hello everyone I am Jennifer and i am excited to be contributing to camicroscope

**Reply by SisiVero** [reply]
Hello Jennifer, you're very welcome.

**Comment by PASEDA-MODUPE** [comment]
Hello everyone I'm Modupe Paseda, I got into Outreachy application phase for may 2024.super excited to contribute to caMicroscope community.Please can you help me understand how to start the Outreachy contribution stage.Thank you

**Reply by SisiVero** [reply]
Hello Modupe, to start the contribution, you'll first of all read the project detials in the caMicriscope landing page which can be found when you click on the link connected to it in the outreachy projects list page. Secondly to commence contribution, you can go through this draft to find details on how to start, here is the link to the draft:
https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md

**Comment by kemsguy7** [comment]
Hello everyone, I am Matthew Idungafa an outreachy applicant and would look forward to contributing to this organization in this internship round

**Reply by gbengaoluwadahunsi** [reply]
Welcome here @kemsguy7

**Reply by SisiVero** [reply]
Hi Matthew, it's good to have you

**Comment by Ayesh-07** [comment]
Hello, I'm Ayesha Nudrat an Outreachy application phase for may 2024 . I'm familiar with HTML, CSS, and Bootstrap, and I'm enthusiastic about learning from the community and mentors. I'm excited to contribute to the "caMicroscope" project.

**Reply by NgweBecky** [reply]
you are welcome

**Reply by SisiVero** [reply]
Hi Ayesha, you're welcome

**Comment by ayush-0110** [comment]
Hello everyone I'm Ayush Tiwari, I got into Outreachy application phase for may 2024. I am very excited to contribute to caMicroscope community. Please can you help me with the steps about how should i start the contribution phase .Thank you

**Reply by NgweBecky** [reply]
hit there and welcome
This was provided by our mentor @birm
"I've put a list of some possible good contributor tasks here (https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md) "

Concerning any suggestions on contributing to the projects documentation, use this link: http://camicroscope.org/docs/

**Reply by SisiVero** [reply]
Hello @ayush-0110 , firstly you need to fork and clone this repo: https://github.com/camicroscope/Distro
Then go through this doc and follow the steps: https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md

**Reply by ayush-0110** [reply]
Thanks

**Comment by oshodiomolola** [comment]
Hello everyone, my name is Oshodi Omolola. outreachy candidate for 2024, and vert excited to contribute to caMicroscope open project

**Reply by DIANAAJAMBO** [reply]
hi @oshodiomolola, you are welcome. To start contributing please  refer to the link below for instructions on how to begin.
https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md

**Comment by robel16** [comment]
Hello, @birm I have sent a patch for issue #699, and can you please review it?
Thank you.
